lib/addons/p5.dom.js

```diff
@@ -112,15 +123,26 @@
    * </code></div>
    *
    */
-  p5.prototype.selectAll = function (e) {
+  p5.prototype.selectAll = function (e, p) {
     var arr = [];
     var res;
     var str;
+    var container = document;
+    if (typeof p === 'string'){
```

Comment:
same comments as above apply here, but seeing this it makes me realize we should probably just have one internal helper function (something like `getParentElement()`) that can be called from both select() and selectAll().